pages/agent/v3/install/_windows.md.erb

```diff
@@ -1,4 +1,4 @@
 1. Download the latest Windows beta from <a href="https://github.com/buildkite/agent/releases">Buildkite Agent releases on GitHub</a>
-2. Extract the files to a directory of your choice
+2. Extract the files to a directory of your choice (we recommend C:\buildkite)
```

Comment:
On linux we use `~/.buildkite-agent` so shouldn't the folder be `Buildkite Agent` or something?

Reply:
Or put it in "UserData"?

Reply:
Possibly. Lots of tools like Git for Windows and Golang go for a simpler approach of a top-level directory that is simply named. When I asked what best practice was from some Windows folks, they indicated a simple top-level dir was preferable. 

Whether it's `C:\buildkite-agent` or `C:\buildkite` is another question. I was going for naming it after the company, but I can see the arguments for either, especially seeing as we aren't namespacing `C:\buildkite\agent` or similar.

Reply:
> Possibly. Lots of tools like Git for Windows and Golang go for a simpler approach of a top-level directory that is simply named. When I asked what best practice was from some Windows folks, they indicated a simple top-level dir was preferable.

Yeah cool, what ever the Windows folks say!

> Whether it's C:\buildkite-agent or C:\buildkite is another question

I'd aim for `buildkite-agent` with parity with the Linux one, and `buildkite-agent` is the name of the software.

Reply:
Ok, I'll PR the agent install script to correct this and then update these docs to reflect it.

Reply:
The linux installer script creates `~/.buildkite-agent/bin` to put the binary in. Do we want to replicate this?